manager/src/main/java/com/redhat/service/smartevents/manager/LimitService.java

```diff
@@ -0,0 +1,7 @@
+package com.redhat.service.smartevents.manager;
+
+import com.redhat.service.smartevents.manager.models.OrganisationServiceLimit;
+
+public interface LimitService {
+    OrganisationServiceLimit getOrganisationServiceLimit(String orgId);
```

Comment:
@vaibhavjainwiz I think this should return a `ServiceLimitInstance` and not the `OrganisationServiceLimit`. From what I can see, `OrganisationServiceLimit` is an internal implementation detail of deriving the end `ServiceLimit`.

Reply:
I change it to return `InstanceLimit`